influxql/call_iterator.go

```diff
@@ -781,39 +781,47 @@ func IntegerSpreadReduceSlice(a []IntegerPoint) []IntegerPoint {
 	return []IntegerPoint{{Time: ZeroTime, Value: max - min}}
 }
 
-func newTopIterator(input Iterator, opt IteratorOptions, n int) (Iterator, error) {
+func newTopIterator(input Iterator, opt IteratorOptions, n int, writeMode bool) (Iterator, error) {
```

Comment:
That sounds fine to me. The name of the variable was rather arbitrary to begin with. I couldn't think of something good.